src/components/Calendar/Event/CreateEvent.js

```diff
@@ -87,38 +98,92 @@ export default function CreateEvent(props) {
     ) : null;
 
     function handleIsRecurrentChange() {
-        setIsRecurrent((prev) =>  !prev);
+        setIsRecurrent((prev) => !prev);
     }
 
     const buttons = (
         <React.Fragment>
-                                   <Stack direction='row' spacing={7} marginTop={2}>
-
-                 <PrimaryButton2  width={'41vw'} colour={'#912338'} content="Add" onClick={handleEvent} />
-                 <PrimaryButton2 width={'41vw'} colour={'#C8C8C8'} content="Cancel" onClick={handleCancel} />
-                 </Stack>
-          </React.Fragment>
+            <Stack justifyContent="center"
+                   alignItems="center"
+                   spacing={3}
+                   width='100%'
+                   direction="row">
+                <PrimaryButton2 width={'41vw'} colour={'#912338'} content="Add" onClick={handleEvent}/>
+                <PrimaryButton2 width={'41vw'} colour={'#C8C8C8'} content="Cancel" onClick={handleCancel}/>
+            </Stack>
+        </React.Fragment>
     );
+    const categories = (
+        <React.Fragment>
+            <Typography style={{fontWeight: 'bold'}}>
+                Select the event category: {eventData.type}
+            </Typography>
+            <Stack direction="row" spacing={2} justifyContent="center">
+                <PrimaryButton2 width={'20vw'} colour={'#0072A8'} content="Course" value="course"
+                                onClick={handleTypeUpdate}/>
+                <PrimaryButton2 width={'20vw'} colour={'#8CC63E'} content="Study" value="study"
+                                onClick={handleTypeUpdate}/>
+                <PrimaryButton2 width={'20vw'} colour={'#DA3A16'} content="Workout" value="workout"
+                                onClick={handleTypeUpdate}/>
+                <PrimaryButton2 width={'20vw'} colour={'#DB0272'} content="Appointment" value="appointment"
+                                onClick={handleTypeUpdate}/>
+            </Stack>
+        </React.Fragment>
+    )
+
+    const course = (
```

Comment:
for const returning JSX elements use the useMemo hook. 
https://beta.reactjs.org/reference/react/useMemo